### PR TITLE
Added Order - Model,Dao,DaoImpl files

### PR DESCRIPTION
**Pull request by sairamch04**
Updated Order-DAO,Models,DaoImpl

### CODE REVIEW

File: src/main/java/com/FoodyBuddy/Dao/Impl/OrderDAOImpl.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    public OrderDAOImpl(Session session){
+    	 this.session = session;
+    }    
+    public void save(Order order) {
+		session.persist(order);
+	}
+	
+	@SuppressWarnings("unchecked")
+	public List<Order> getListByBuyerId(Integer buyerId){
+		String query= "FROM Order O WHERE O.buyer_id = " + buyerId;
```
Comment (junzy) [inline src/main/java/com/FoodyBuddy/Dao/Impl/OrderDAOImpl.java]: No need to assign Order as O. Not solving any ambiguity

File: src/main/java/com/FoodyBuddy/Dao/Impl/OrderDAOImpl.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
+package com.FoodyBuddy.Dao.Impl;
```
Comment (kanmaytacker) [inline src/main/java/com/FoodyBuddy/Dao/Impl/OrderDAOImpl.java]: Dao in package should be all capitals i.e. DAO. Inconsistent with your own naming convention.

File: src/main/java/com/FoodyBuddy/Model/Order.java [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+	@Column(name="id")
+	@GeneratedValue
+	private int id;
+	
+	
+	@Column(name="name" , nullable = false)
+	private String name;
+	
+	@Column(name = "buyer_id", nullable = false)
+    @ManyToOne
```
Comment (kanmaytacker) [inline src/main/java/com/FoodyBuddy/Model/Order.java:27]: Indentation discrepancy.

File: src/main/java/com/FoodyBuddy/Dao/OrderDAO.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import java.util.List;
+
+import com.FoodyBuddy.Model.Order;
+
+public interface OrderDAO {
+
+	public void save(Order c);
+	public List<Order> getListByBuyerId(Integer buyerId);
+	public Order getByOrderId(Integer orderId);	
+	public Order getById(Integer id);
```
Comment (kanmaytacker) [inline src/main/java/com/FoodyBuddy/Dao/OrderDAO.java]: Ambiguous Function Name.

File: src/main/java/com/FoodyBuddy/Dao/Impl/OrderDAOImpl.java [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+		return orders;		
+	}
+	@SuppressWarnings("unchecked")
+	public Order getByOrderId(Integer orderId){
+		String query= "FROM Order O WHERE O.id = " + orderId;
+		Order order = (Order) session.createQuery(query).list().get(0);
+		return order;
+		
+	}
+
```
Comment (kanmaytacker) [inline src/main/java/com/FoodyBuddy/Dao/Impl/OrderDAOImpl.java:32]: Ambiguous Function.

File: src/main/java/com/FoodyBuddy/Dao/Impl/OrderDAOImpl.java [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+		return order;
+	}
+	
+	public void update(Order order) {
+		session.update(order);	
+	}
+
+	public void delete(Order  order) {
+		session.delete(order);
+
```
Comment (kanmaytacker) [inline src/main/java/com/FoodyBuddy/Dao/Impl/OrderDAOImpl.java]: Inconsistent Line spacing.

File: src/main/java/com/FoodyBuddy/Dao/OrderDAO.java [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+	public void update(Order order);	
+	public void delete(Order order);
```
Comment (SainathB) [inline src/main/java/com/FoodyBuddy/Dao/OrderDAO.java:13]: While Implementing Buyer, we deleted an Object using id. So we need to come up with a uniform solution.

**Pull request merged**